app/styles/vars.scss

```diff
@@ -1,3 +1,3 @@
-$page-top-margin: 20px;
+$page-top-margin: 14px;
```

Comment:
@anu-007 won't this change affect other areas which were not to be changed as well? Can you please confirm?